_posts/articles/2022-08-16-genart.md

```diff
@@ -0,0 +1,28 @@
+
+# AI Village Art Day @ DEF CON 30
+Today, models that appear to generate novel outputs conditioned on text are becoming the mainstay of popular culture.
+New groups of users are starting to adopt generative architectures and bring new modes (experiential, creative) of interacting with large models.
+The understanding of model performance is becoming increasingly qualitative and subjective, as is evident in the latest benchmarks for text-to-image models.
+
+We would like to give a platform to the artists and those building large foundation models to come together, and begin to form a shared understanding of each other's approaches.
+## Context
```

Comment:
The below should be in paragraph or bulleted form, as it is this is messy and unpolished/unfinished